immunity/demo_immunity_enrol.py

```diff
@@ -11,16 +11,23 @@
 
 data = {
     'api_secret': 'e1131458-4664-4da7-855a-7ac3e5b9648d',
-    'first_name': 'ANGELA ZOE',
-    'last_name': 'UK SPECIMEN',
-    'issue_date': '1592310965',
-    'expiry_date': '1592310966',
-    'selfie': selfie_base64_encoded,
+    'identity': {
+        'first_name': 'ANGELA ZOE',
+        'last_name': 'UK SPECIMEN',
+        'selfie_base64_encoded': selfie_base64_encoded,
+        'email_address': 'angela.zoe@gov.uk', # optional
+        'phone_number': '00447700123456',     # optional
+        'date_of_birth': '19881204'           # optional
+    },
     'test': {
```

Comment:
@robin-ankele Can you comment here whether these are optional or mandatory?

Reply:
Actually, for all keys that are optional, comment # option 
If nothing is mentioned, let's assume its mandatory